### PR TITLE
Adjust tests to new fixtures

### LINKED ISSUE #3: A test flaking
After this PR it's passing locally but failing on one test in CI https://github.com/ssbc/ssb-search2/pull/2 (not sure what was the status before the PR, it couldn't install for me)

### PR DESCRIPTION
**Pull request by Powersource**
Fixes https://github.com/ssbc/ssb-search2/issues/3

So I did https://github.com/ssbc/ssb-search2/pull/2/files and there were failing tests in ci but i was like pfft silly CI. Turns out my machine didn't get the new, different fixtures (had to upgrade ssb-fixtures because of rsjs). So this PR adjusts the tests to actually pass with the new fixtures

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ test('exact chain-of-words', (t) => {
     pull.filter((msg) => msg.value.content.text.toLowerCase().includes(INPUT)),
     pull.collect((err, msgs) => {
       t.error(err, 'no error');
-      t.equals(msgs.length, 4, 'four posts');
+      t.equals(msgs.length, 2, 'two posts');
```
Comment (Powersource) [inline test/index.js:72]:
```bash
% rg -a 'labore officia exercitation' /tmp/ssb-search2/flume/log.offset|wc -l
2
```
(there are multiple messages on each line but i verified by looking at the results in `less`)

Comment (Powersource) [reply test/index.js:72]: tried finding a search with a few more results but i could only find searches with either 1-2 results or like 30+ (which would be harder to verify)

File: test/index.js [hunk 2]
```diff
@@ -91,9 +91,9 @@ test('word prefix', (t) => {
       t.false(/\blabo\b/i.test(msg.value.content.text), 'labo is not found');
       const matches = [...msg.value.content.text.matchAll(/labo/gi)];
       t.true(msg.value.content.text.includes('labo'), 'labo prefix is found');
-      t.equals(matches.length, 3, '3 matches');
+      t.equals(matches.length, 2, '2 matches');
       const matchPositions = matches.map((m) => m.index);
-      t.deepEquals(matchPositions, [47, 55, 168], 'labo positions found');
+      t.deepEquals(matchPositions, [160, 168], 'labo positions found');
```
Comment (Powersource) [inline test/index.js:96]: i think this is the message it's running on

```json
{"key":"%rEpV0vnOoBuO01jmBb90H4Af/FmxTasGV0+otihgN8s=.sha256","value":{"previous":null,"sequence":1,"author":"@J74J+ctsqJNREBjVqvc/oeSVWNRrGcrfH54ERGQtqwI=.ed25519","timestamp":17059425
45831,"hash":"sha256","content":{"type":"post","text":"OLDESTMSG reprehenderit officia ad","c
hannel":"mollit","mentions":[{"link":"@3rJVBV5zDW8wgCEjAHBgNAkiE2c+HJf2XIsU6aND3Z8=.ed25519","name":"laborum laboris reprehenderit"},{"link":"&ZGVzZXJ1bnRsYWJvcmVvY2NhZWNhdGR1aXNuaXNpZXg=.sha256","type":"image/png","size":1220163},{"link":"@n7mkSt+3MKjSP5S8R6lJx2ixZKefldZ8GuyyBbtNCY4=.ed25519","name":"magna est"},{"link":"@QxRm+ZF6ZmZtMRDbdTvSL5FVG8XjiP0m5qHqRnSIacI=.ed25519","name":"nostrud"}]},"signature":"BAiI3JmQM7ZVkZbQdvLvHIFTgM2zltmUHzcxRF9kkQpLPcKvSTbbaH67Hl0i1XHBgb5vzNUltDoV57iRMjzaAA==.sig.ed25519"},"timestamp":1705942545832}
```

the matching text seems to be at sort of 160 and 168 chars into the `content` so it's probably ok? this is a bit more of a hunch on my part, idk exactly what `.index` is.

Comment (staltz) [reply test/index.js:96]: I don't know either what's going on, but it's fine by me :shrug:

**Review by staltz — APPROVED** [review]
Looks good :)

You can now merge yourself, and npm publish (although in this case not applicable).

**Pull request merged**